### PR TITLE
[chore] move prometheusremotewrite exporter to generated lifecycle tests

### PR DESCRIPTION
**Pull request by Frapschen**
Relates to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849

### CODE REVIEW

File: exporter/prometheusremotewriteexporter/metadata.yaml [hunk 1]
```diff
@@ -7,3 +7,8 @@ status:
   distributions: [core, contrib, aws, observiq, liatrio]
   codeowners:
     active: [Aneurysm9, rapphil]
+
```
Comment (crobert-1) [inline exporter/prometheusremotewriteexporter/metadata.yaml:10]:
```suggestion
```

Just a nit, one empty line is probably fine here 👍

**Review by atoulme — APPROVED** [review]
LGTM

**Review by codeboten — COMMENTED** [review]
Pinging code owners @Aneurysm9 @rapphil

**Review by bryan-aguilar — APPROVED** [review]

**Review by Aneurysm9 — APPROVED** [review]

**Comment by bryan-aguilar** [comment]
`Generated code is out of date, please run "make generate" and commit the changes in this PR.`
@Frapschen

**Review by rapphil — APPROVED** [review]

**Review by evan-bradley — APPROVED** [review]

**Pull request merged**